app/dashboard/models.py

```diff
@@ -377,6 +393,12 @@ class Bounty(SuperModel):
     value_in_usdt = models.DecimalField(default=0, decimal_places=2, max_digits=50, blank=True, null=True)
     value_in_eth = models.DecimalField(default=0, decimal_places=2, max_digits=50, blank=True, null=True)
     value_true = models.DecimalField(default=0, decimal_places=2, max_digits=50, blank=True, null=True)
+
+    usd_pegged_value_in_token_now = models.DecimalField(default=0, decimal_places=2, max_digits=50, blank=True, null=True) # The calculated amount in token, corresponding to value_true_usd
+    usd_pegged_value_in_token = models.DecimalField(default=0, decimal_places=2, max_digits=50, blank=True, null=True)     # The calculated amount in token, corresponding to value_true_usd
+    value_true_usd = models.DecimalField(default=0, decimal_places=2, max_digits=50, blank=True, null=True)     # The value the user wants to pay in USD
+    peg_to_usd = models.BooleanField(default=False)                                                             # True if the amount to pay should be pegged to USD
```

Comment:
^ can none of the above fields like value_in_usdt etc be resued ? Just curious

Reply:
This is probably possible, but I think having this additional variables makes following the logic flow much easier:
- if the user does not peg to USD -> we use the old vars, and conversions are made from token amount -> usd amount
- if the user does peg to USD -> we use the new vars, and conversions are made from usd amount -> token amount

I think this helps understand better which value the user entered:  value_true respectively value_true_usd.